src/NumericalAlgorithms/Interpolation/Actions/InterpolateToTarget.hpp

```diff
@@ -1,136 +0,0 @@
-// Distributed under the MIT License.
```

Comment:
Actually, sorry @wthrowe , I've thought of a case that I'm not sure how to handle without something similar to this action.
I'll need to send data to CCE during self-start, and I haven't been able to think of a good way of doing that without requiring that the two systems have identical steppers and slabs during self start, and then sending the boundary data from GH on every time step so that the matched time steps in CCE can be evaluated.

So, as peculiar as it is, I think what I actually will need is basically this action converted to just send at `time_id.substep_time().value()`, and to be used exclusively during self-start.
All this seems pretty inconvenient to do for this PR, so unless you particularly want to make the conversions, I'll just add back a version of this action that works for my use case in a separate PR in which i work out all the details of using dense output for the CCE interpolation data.

Reply:
Alright, I'll let you handle that.  I have no objection to the action coming back in some form if it is useful, and I can't think of a way around it during self-start either.